### DISCUSSION TITLE
Documentation going to 404

### DISCUSSION THREAD

**Discussion by Mufassa4n6** [google/timesketch#2010] [Q&A, quick solutions, support] [answered]
Hello, I am interested in setting up a timesketch server but all the documentation is going to 404 pages. Is there anywhere else that we can get the documentation?

**Comment by jaegeral** [comment, marked as answer]
Hi, ths for flagging, the links should be fixed now:

https://timesketch.org/guides/admin/install/

with https://github.com/google/timesketch/pull/2017
